components/slider-carousel/src/ReuseSliderCarousel.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { useState, useRef, useEffect } from "react";
+import { useSlider } from "@locoworks/reusejs-toolkit-react-hooks";
+import { twMerge } from "tailwind-merge";
+
+interface ReuseSliderCarouselInterface {
+  slideInterval: number;
+  slides: React.ReactNode[];
+  loop: boolean;
+  wrapperClasses?: string;
+  sliderContainerClasses?: string;
+  enableButtons?: boolean | false;
+  prefixButton?: React.ReactNode;
+  suffixButton?: React.ReactNode;
+  buttonClasses?: string;
```

Comment:
Are these styles? Will it accept CSS values if needed?

Reply:
Yes, User able use Tailwind classes directly or they can pass there custom classes name. this totally upto the user how he want to style it. 

also user can pass the button as component if Default button needs to be replaced.

Reply:
If these button component can be replaced if needed by the user, than why even give the ability to change the styles. If we are giving the prop to change styles means we allow customisation. So a user should be able to customise using both `css` and `tailwind`.

Reply:
Thank You got the point, 
I removed buttonClasses as this has no relevance here.

The code now includes default buttons. However, users can replace them with their own buttons by passing button components via the `prefixButton` and `suffixButton` props.